README.md

```diff
@@ -30,7 +32,7 @@ We expect members of the community participating in conversations here to act in
 - Give constructive feedback to ideas
 - Search the issue tracker to ensure that your issue has not already been submitted before making a new post (use the search bar on the [Issues page](https://github.com/Minehut/Meta/issues))
 - Verify the validity of your issue
-- Be clear, concise and specific in your messages
+- Be clear, concise, and specific in your messages
```

Comment:
This is following the US rules, considering UK rules, the previous message was correct.

Reply:
I don't think that follows US/UK rules. Adding a comma before `and` is an optional thing to do in writing, it's especially used when listing lengthy items that may contain the term `and` (I think, correct me if I'm wrong). I don't really mind this change, tbh I think it looks nicer.

Reply:
I don't think that follows US/UK rules. Adding a comma before `and` is an optional thing to do in writing, it's especially used when listing lengthy items that may contain the term `and` (I think, correct me if I'm wrong). I don't really mind this change, tbh I think it looks nicer.